docs/support/logging/Ch-Logging.rst

```diff
@@ -46,44 +44,7 @@ High Level Design Architecture
 
 .. image:: EdgeX_SupportingServicesLoggingArchitecture.png
 
-The above diagram shows the high-level architecture for EdgeX Foundry Logging Service, which uses the Spring Boot Application Framework. Other microservices interact with EdgeX Foundry Logging Service through RESTful APIs to submit their logging requests, query historical logging, and remove historical logging. Internally, EdgeX Foundry Logging Service utilizes LOGBack as its underneath logging framework. Two configurable persistence options exist supported by EdgeX Foundry Logging Service: file or MongoDB. 
-
-==========
-Data Model
-==========
-
-.. image:: EdgeX_SupportingServicesLoggingDataModel.png
-
-
-===============
-Data Dictionary
-===============
-
-+---------------------+--------------------------------------------------------------------------------------------+
-|   **Class Name**    |   **Descrption**                                                                           | 
-+=====================+============================================================================================+
-| LogEntry            | The object describing a particular log message including origin, severity, and content.    | 
-+---------------------+--------------------------------------------------------------------------------------------+
-| MatchCriteria       | The object describing the search parameters for a log query.                               | 
-+---------------------+--------------------------------------------------------------------------------------------+
-
-===============================
-High Level Interaction Diagrams
-===============================
-
-This section shows the sequence diagrams for EdgeX Foundry Logging Service.
-
-**Sequence Diagram for Logging Request**
-
-.. image:: EdgeX_SupportingServicesLoggingRequest.png
-
-**Sequence Diagram for Query Historical Logging**
-
-.. image:: EdgeX_SupportingServicesQueryHistoricalLogging.png
-
-**Sequence Diagram for Removing Historical Logging**
-
-.. image:: EdgeX_SupportingServicesRemoveHistoricalLogging.png
+The above diagram shows the high-level architecture for EdgeX Foundry Logging Service. Other microservices interact with EdgeX Foundry Logging Service through RESTful APIs to submit their logging requests, query historical logging, and remove historical logging. Internally, EdgeX Foundry's Logging Service utilizes the .. _GoKit logger: https://github.com/go-kit/kit/tree/master/log as its internal logging framework. Two configurable persistence options exist supported by EdgeX Foundry Logging Service: file or MongoDB.
```

Comment:
Hyperlinks need to be defined as:

\`Link Text \<URL\>\`_

So this should be:

\`GoKit logger \<https://github.com/go-kit/kit/tree/master/log>`_

Reply:
Change was made